bundle/edu.gemini.auxfile.workflow/src/main/java/edu/gemini/auxfile/workflow/CheckTask.java

```diff
@@ -17,7 +17,11 @@ public CheckTask(SPProgramID progId, File file, boolean checked) {
 	@Override
 	public void execute(Workflow wf) throws Exception {
         final AuxFileType type = AuxFileType.getFileType(getFile());
-        wf.getMailer().notifyChecked(getProgId(), type, getFile().getName(), _checked);
+
+        // Try to send out emails if notification for this file type is desired.
+        if (type.sendNotification()) {
```

Comment:
Not sure what the question is. This is not `notify()` or some such thing but just a `boolean` that tells you if emails should be sent out for this aux file type or not. Maybe the name is misleading and it should be renamed to `sendEmails()`?